avocado/core/nrunner.py

```diff
@@ -557,29 +557,32 @@ class PythonUnittestRunner(BaseRunner):
 
      * kwargs: not used
     """
-    @staticmethod
-    def _uri_to_unittest_name(uri):
+
+    @property
+    def unittest_name(self):
+        """Convert a test reference (uri) to an unittestname reference."""
```

Comment:
Missing space between `unittest` and `name`?

Reply:
I had fixed this but on the wrong commit. thanks.